public/app/features/search/components/SearchItem.tsx

```diff
@@ -43,7 +52,22 @@ export const SearchItem: FC<Props> = ({ item, editable, onToggleChecked, onTagSe
       <Card.Figure align={'center'}>
         <SearchCheckbox editable={editable} checked={item.checked} onClick={toggleItem} />
       </Card.Figure>
-      {item.folderTitle && <Card.Meta>{item.folderTitle}</Card.Meta>}
+      {(item.folderTitle || item.sortMeta) && (
+        <Card.Meta separator={''}>
+          {item.folderTitle && (
```

Comment:
I think it would be nice to also have the folder name displayed when the dashboard is in the "General" folder. We could handle it here or in the backend. I looked a bit into how to do it in the backend, the change is easy but we need to make sure that there is no impact on other features... WDYT?

Reply:
This could be done on front end, where we group the dashboards into General folder. I think it's a good idea to display General folder name, although not sure if some users might get confused. @torkelo any ideas?

Reply:
yea showing General sounds like a good idea